### PR TITLE
(role/htcondor) installs htcondor scheduler

### PR DESCRIPTION
**Pull request by cbarria**
HTCondor Scheduler profile

### CODE REVIEW

**Comment by jhoblitt** [comment]
@cbarria Do you want review on this PR while its a draft and (WIP)?

**Comment by cbarria** [comment]
not yet, i might noodle with github in the wrong way today... we have the module class still pending, sorry.

**Comment by jhoblitt** [comment]
@cbarria FYI - Even if a PR is set as a draft, when you add reviewers the request shows up in https://github.com/pulls/review-requested.

File: hieradata/site/cp/role/htcondor.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+classes:
```
Comment (jhoblitt) [inline hieradata/site/cp/role/htcondor.yaml]: It appears that `spec/unit/top_level_role_spec.rb` was broken by the migration to eyaml.  It should have failed on top level `htcondor` role not existing.  Our pattern is that there must be a top level role.  The classes/etc. should be factored out to a common top level role.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/site/cp/role/htcondor.yaml (hunk 1 above)
Comment (jhoblitt) [reply hieradata/site/cp/role/htcondor.yaml]: Fix for the broken spec: https://github.com/lsst-it/lsst-control/pull/1118

Comment (cbarria) [reply hieradata/site/cp/role/htcondor.yaml]: I need more specifics on this as I don't fully understand what it means, I will ask later, thanks!

Comment (jhoblitt) [reply hieradata/site/cp/role/htcondor.yaml]: `hieradata/role/htcondor.yaml` is required to exist.

Comment (jhoblitt) [reply hieradata/site/cp/role/htcondor.yaml]: #1118 has been merged.  When this branch is rebased on `production`, there should be a test failure now.

Comment (cbarria) [reply hieradata/site/cp/role/htcondor.yaml]: yeah it broke but i added the file and now its fine 👍🏻

File: hieradata/site/cp/role/htcondor.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+htcondor::htcondor_version: "23.0"
```
Comment (jhoblitt) [inline hieradata/site/cp/role/htcondor.yaml]: It looks like the version can be factored out to the top level class too.

Comment (cbarria) [reply hieradata/site/cp/role/htcondor.yaml]: your right, versions should stay the same and if it needs a different on dev, it can be overwritten at site

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**